app/src/main/java/com/github/dedis/student20_pop/ui/AttendeeFragment.java

```diff
@@ -38,16 +36,14 @@
     private Button propertiesButton;
     private ListView witnessesListView;
 
-
-    @RequiresApi(api = Build.VERSION_CODES.O)
     @Nullable
     @Override
     public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
 
         View rootView = inflater.inflate(R.layout.fragment_attendee, container, false);
         PoPApplication app = (PoPApplication) (this.getActivity().getApplication());
-        lao = app.getCurrentLao();
-        List<Event> events = app.getEvents(lao);
+        lao = app.getCurrentLaoUnsafe();
```

Comment:
Why do you use here the `getCurrentLaoUnsafe` instead of the `getCurrentLao` method?